docs/build/micro-rdk/motor/gpio.md

```diff
@@ -51,14 +51,15 @@ Edit and fill in the attributes as applicable.
       "namespace": "rdk",
       "attributes": {
         "pins": {
-          <...>
+          "dir": "<int>",
+          "pwm": "<int>",
+          "en_low": "<int>"
```

Comment:
actually we should just remove `en_low`, we're not going to support this in micro-RDK (also applies to L. 136 and L. 143 below)

Reply:
Ok sounds good!